lib-ee/emqx_ee_bridge/src/kafka/emqx_bridge_impl_kafka_consumer.erl

```diff
@@ -145,6 +145,16 @@ on_start(InstanceId, Config) ->
                 instance_id => InstanceId,
                 kafka_hosts => BootstrapHosts
             });
+        {error, already_present = Reason} ->
+            ?SLOG(error, #{
+                msg => "failed_to_start_kafka_consumer_client",
+                instance_id => InstanceId,
+                kafka_hosts => BootstrapHosts,
+                reason => emqx_misc:redact(Reason)
+            }),
+            throw(
+                "Failed to start Kafka client. Please check the connection parameters."
```

Comment:
are you sure this throw is added to the right clause ?
it seems to me this clause means the client is already running.?

Reply:
I tested the reported scenario: the wrong port is used, so the bridge is created, but the client never gets to connect.  In such state, the user clicks "Reconnect" in the UI, which [internally means](https://github.com/emqx/emqx/blob/9b35aeec6da31c7ab93e770ffa5334f7efb5a65d/apps/emqx_bridge/src/proto/emqx_bridge_proto_v3.erl#L97-L106) to start the bridge again.

In that situation, we get `already_started` because the brod client process indeed started, as we're starting the same bridge with the same name and thus kafka client id, but keeps crashing while trying to connect to Kafka.

Reply:
I've added a couple screenshots in the description to illustrate the before and after errors messages.